projects/go-lib/src/lib/components/go-portal/go-portal.service.ts

```diff
@@ -0,0 +1,31 @@
+import {Injectable, TemplateRef, ViewContainerRef} from '@angular/core';
+
+@Injectable({providedIn: 'root'})
```

Comment:
Should we throw `providedIn: 'root' on a new line or add some spaces around the `{ }` here?